study_hour/react-client/src/Components/App.jsx

```diff
@@ -66,6 +67,9 @@ class App extends Component {
                         <Route path="/Location/:id" component={Location}/>
                         <Route path="/Location" component={Location}/>
                         <Route path="/AddLocation" component={AddLocation}/>
+                        <Route path="/ValidateUser" component={ValidateUser}/>
+                        <Route path="/SecurityCheck" component={SecurityCheck}/>
+                        <Route path="/ResetPassword" component={ResetPassword} />
```

Comment:
can we wrap NoAuth components in NoAuth wrapper?

Reply:
LOOK!